scrapy/downloadermiddlewares/retry.py

```diff
@@ -40,19 +38,25 @@ class RetryMiddleware:
                            ConnectionLost, TCPTimedOutError, ResponseFailed,
                            IOError, TunnelError)
 
+    SETTINGS_PREFIX = ""
+    META_PREFIX = ""
+    STATS_PREFIX = ""
+    LOGGER_NAME = __name__
```

Comment:
Maybe we could allow to define a custom `logger` object, rather than a name, for more flexibility?